crawl-ref/source/webserver/game_data/static/game.js

```diff
@@ -225,13 +226,32 @@ function ($, comm, client, dungeon_renderer, display, minimap, enums, messages,
         document.title = data.text;
     }
 
-    var device_ratio = window.devicePixelRatio;
-    var renderer_settings = {
-        glyph_mode_font_size: 24 * device_ratio,
-        glyph_mode_font: "monospace"
-    };
+    function glyph_mode_font_init()
+    {
+        if (options.get("tile_display_mode") == "tiles") return;
+
+        var device_ratio = window.devicePixelRatio;
+        var glyph_font, glyph_size;
+
+        if (!options.get("glyph_mode_font_size"))
+            glyph_size = 24 * device_ratio;
+        else
+            glyph_size = options.get("glyph_mode_font_size") * device_ratio;
```

Comment:
Are these guards necessary? The web tiles server should send out the whole of the "Options" object, with the defaults included.